### PR TITLE
[BEAM-13881] [Playground] Increase test coverage for the utils package

### PR DESCRIPTION
**Pull request by pavel-avilov**
[[BEAM-13881]](https://issues.apache.org/jira/browse/BEAM-13881)
Added unit tests to increase test coverage for the utils package.

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/#make-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

File: playground/backend/internal/utils/common_test.go [hunk 1]
```diff
@@ -15,7 +15,51 @@
...
+	err := setup()
+	if err != nil {
+		panic(fmt.Errorf("error during test setup: %s", err.Error()))
+	}
+	defer teardown()
+	m.Run()
+}
+
+func setup() error {
+	err := os.Mkdir(sourceDir, 0755)
```
Comment (AydarZaynutdinov) [inline playground/backend/internal/utils/common_test.go:47]: `0755` as a constant

File: playground/backend/internal/utils/preparares_utils_test.go [hunk 2]
```diff
@@ -50,3 +60,347 @@ func TestSpacesToEqualsOption(t *testing.T) {
...
 }
+
+func TestInitVars(t *testing.T) {
+	tests := []struct {
+		name  string
+		want  string
+		want1 string
+		want2 error
+		want3 bool
+		want4 PipelineDefinitionType
```
Comment (AydarZaynutdinov) [inline playground/backend/internal/utils/preparares_utils_test.go:71]: maybe change this one with a slice?

File: playground/backend/internal/utils/preparares_utils_test.go [hunk 3]
```diff
@@ -50,3 +60,347 @@ func TestSpacesToEqualsOption(t *testing.T) {
...
+		spaces       *string
+		regs         *[]*regexp.Regexp
+		tempFile     *os.File
+		err          error
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    bool
+		want1   PipelineDefinitionType
```
Comment (AydarZaynutdinov) [inline playground/backend/internal/utils/preparares_utils_test.go:300]: try do not to use `want, want1` naming.

**Review by AydarZaynutdinov — CHANGES_REQUESTED** [review]

File: playground/backend/internal/utils/common_test.go (hunk 1 above)
Comment (pavel-avilov) [reply playground/backend/internal/utils/common_test.go]: Done

File: playground/backend/internal/utils/preparares_utils_test.go (hunk 2 above)
Comment (pavel-avilov) [reply playground/backend/internal/utils/preparares_utils_test.go]: Done

File: playground/backend/internal/utils/preparares_utils_test.go (hunk 3 above)
Comment (pavel-avilov) [reply playground/backend/internal/utils/preparares_utils_test.go]: Done.

**Review by AydarZaynutdinov — APPROVED** [review]
LGTM

File: playground/backend/internal/utils/preparares_utils_test.go [hunk 4]
```diff
@@ -50,3 +60,328 @@ func TestSpacesToEqualsOption(t *testing.T) {
...
+		newLine bool
+		file    *os.File
+	}
+	tests := []struct {
+		name    string
+		args    args
+		wantErr bool
+	}{
+		{
+			name: "With newLine = false",
```
Comment (daria-malkova) [inline playground/backend/internal/utils/preparares_utils_test.go:225]: Can be renamed like "No line added to file"

File: playground/backend/internal/utils/preparares_utils_test.go [hunk 5]
```diff
@@ -50,3 +60,328 @@ func TestSpacesToEqualsOption(t *testing.T) {
...
+	}
+	tests := []struct {
+		name        string
+		args        args
+		want        bool
+		wantDefType PipelineDefinitionType
+		wantErr     bool
+	}{
+		{
+			name: "Empty curLine",
```
Comment (daria-malkova) [inline playground/backend/internal/utils/preparares_utils_test.go:285]: Empty current line (like below)

**Review by daria-malkova — CHANGES_REQUESTED** [review]

File: playground/backend/internal/utils/preparares_utils_test.go (hunk 4 above)
Comment (pavel-avilov) [reply playground/backend/internal/utils/preparares_utils_test.go:225]: Done

File: playground/backend/internal/utils/preparares_utils_test.go (hunk 5 above)
Comment (pavel-avilov) [reply playground/backend/internal/utils/preparares_utils_test.go]: Done

**Review by daria-malkova — APPROVED** [review]
LGTM

**Comment by pavel-avilov** [comment]
R: @pabloem

**Comment by pabloem** [comment]
thanks LGTM!

**Pull request merged**